gc/base/ParallelDispatcher.cpp

```diff
@@ -757,4 +754,61 @@ MM_ParallelDispatcher::expandThreadPool(MM_EnvironmentBase *env)
 
 	return result;
 }
+
+bool
+MM_ParallelDispatcher::reinitialize(MM_EnvironmentBase *env, uintptr_t newPoolSize)
```

Comment:
append the name so it's more descriptive (that it's about the thread table/pool)